bower.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "object-path",
-  "version": "0.9.2",
```

Comment:
naw just remove it https://github.com/bower/bower.json-spec/tree/0b34d2bb0bdd4fa31fa9728f787f49e170e41f99#version